### PR TITLE
See Google Cal Schedule

### LINKED ISSUE #20: Write function to get schedule from calendar
Update task_creation function in task_command.py
Update habit_creation function in habit_command.py

### PR DESCRIPTION
**Pull request by juliussneezer04**
- Adds functions to see google cal schedule for user
- Fixes #20

### CODE REVIEW

**Comment by juliussneezer04** [comment]
@joshuan98 need help to see if schedule is used as intended in `habit_command.py`

File: commands/habit_command.py [hunk 1]
```diff
@@ -62,7 +62,15 @@ async def habit_creation(update: Update, context: ContextTypes.DEFAULT_TYPE):
...
-    # TODO: get schedule from calendar
+    user = context.user_data or {} # TODO Get User from DB
+    time_min, time_max = get_datetimes_till_end_of_day()
+    cal_schedule_events_str = get_readable_cal_event_string(
+        get_calendar_events(
+        refresh_token=user.get("google_refresh_token", None),
+        timeMin=time_min.isoformat(),
+        timeMax=time_max.isoformat(),
+        k=15,
+    ))
```
Comment (juliussneezer04) [inline commands/habit_command.py:73]: this is the first place

File: commands/task_command.py [hunk 2]
```diff
@@ -91,8 +94,15 @@ async def task_schedule_yes_update(update, context):
...
-    schedule = "<schedule>"
+    user = context.user_data or {} # TODO Get User from DB
+    time_min, time_max = get_datetimes_till_end_of_day()
+    cal_schedule_events_str = get_readable_cal_event_string(
+        get_calendar_events(
+        refresh_token=user.get("google_refresh_token", None),
+        timeMin=time_min.isoformat(),
+        timeMax=time_max.isoformat(),
+        k=15,
+    ))
```
Comment (juliussneezer04) [inline commands/task_command.py:105]: second

Comment (dZhei98) [reply commands/task_command.py:105]: Yes, correct

File: commands/habit_command.py (hunk 1 above)
Comment (dZhei98) [reply commands/habit_command.py:73]: This doesn't need a readable string, just needs events but yes it's in the correct place.

**Pull request merged**